Improve wording in README (#324)
kowainik/tomland: README.md

    }

data User
    = Guest !Integer  -- id of guest
    | Registered !RegisteredUser  -- login and createdAt of registered user

data RegisteredUser = RegisteredUser
    { registeredUserLogin     :: !Text
    , registeredUserCreatedAt :: !Day
    }

newtype Port = Port Int
newtype Host = Host Text
```

Using the `tomland` library, you can write bidirectional converters for these types

> using the following guidelines and helper functions:

Reviewer: ```suggestion
with the following guidelines and helper functions:
```

> 1. If your fields are some simple basic types like `Int` or `Text` you can just

Reviewer: ```suggestion
1. If your fields are some simple primitive types like `Int` or `Text` you can just
```

   use standard codecs like `Toml.int` and `Toml.text`.
2. If you want to parse `newtype`s, use `Toml.diwrap` to wrap parsers for
   underlying `newtype` representation.

> 3. For parsing nested data types, use `Toml.table`. But this requires to specify

Reviewer: ```suggestion
3. For parsing nested data types, use `Toml.table`. But it requires to specify
```

   this data type as TOML table in the `.toml` file.
4. If you have lists of custom data types, use `Toml.list`. Such lists are

>    represented as array of tables in TOML. If you have lists of primitive types

Reviewer: ```suggestion
   represented as _array of tables_ in TOML. If you have lists of the primitive types
```

    <$> Toml.diwrap (Toml.text "host") .= mailHost
    <*> Toml.bool "send-if-inactive"   .= mailSendIfInactive

matchGuest :: User -> Maybe Integer
matchGuest = \case
   Guest i -> Just i
   _ -> Nothing

matchRegistered :: User -> Maybe RegisteredUser
matchRegistered = \case
   Registered u -> Just u
   _ -> Nothing

userCodec :: TomlCodec User
userCodec =
        Toml.dimatch matchGuest      Guest      (Toml.integer "guestId")
    <|> Toml.dimatch matchRegistered Registered registeredUserCodec

registeredUserCodec :: TomlCodec RegisteredUser
registeredUserCodec = RegisteredUser
    <$> Toml.text "login"     .= registeredUserLogin
    <*> Toml.day  "createdAt" .= registeredUserCreatedAt
```

> And now we're ready to parse our TOML and print the result back to see whether

Reviewer: ```suggestion
And now we are ready to parse our TOML and print the result back to see whether
```
